tests/backends/storage/test_swift.py

```diff
@@ -105,11 +104,11 @@ def mock_successful_stat(*args, **kwargs):  # pylint:disable=unused-argument
     swift = swift()
     msg = "Failed to list container ralph_logs_container: Container not found"
     with pytest.raises(BackendException, match=msg):
-        assert list(swift.list()) == []
+        list(swift.list())
```

Comment:
There is no need to cast this to a list. We only need to call the `list()` method.

Reply:
As the `list()` method returns a generator, we agreed to use `next()` instead.